### PR TITLE
[Term Entry] PHP Arrays range()

### PR DESCRIPTION
**Pull request by EspressoCat**
### Description

- Added range() term for PHP
- Closes issue #2754 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/arrays/terms/range/range.md
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/range/range.md:3]:
```suggestion
Description: 'Returns an array that contains a series of elements based on the parameters given.'
```

File: content/php/concepts/arrays/terms/range/range.md
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/range/range.md:15]:
```suggestion
The **`range()`** function generates an array containing a sequence of elements. The function allows for the specification of a starting point, an endpoint, and an optional step value to determine the increment between elements.
```

File: content/php/concepts/arrays/terms/range/range.md
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/range/range.md:29]:
```suggestion
The `range()` function returns an array of elements from `$start` to `$end`, and is inclusive (the `$end` value will be the final element in the array).
```

File: content/php/concepts/arrays/terms/range/range.md
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/range/range.md:60]:
```suggestion
// Printing elements of the array
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@EspressoCat I've added my initial comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by EspressoCat** [comment]
Thanks for the comments @caupolicandiaz
made requested changes.

**Review by caupolicandiaz — APPROVED** [review]
@EspressoCat This is ready for a second review.

**Comment by EspressoCat** [comment]
hey @codecademy-docs 👋🏻

Let me know if addition changes are required. Thanks!

**Comment by caupolicandiaz** [comment]
> hey @codecademy-docs 👋🏻
> 
> Let me know if addition changes are required. Thanks!

We appreciate your eagerness and interest, however, please be patient. Another maintainer will conduct a review when they can.

**Review by SSwiniarski — APPROVED** [review]
@EspressoCat Looks good.

**Comment by SSwiniarski** [comment]
@EspressoCat Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/php/arrays/range

image

**Pull request merged**
